addons/magazinerepack/stringtable.xml

```diff
@@ -100,7 +100,7 @@
         <Key ID="STR_ACE_MagazineRepack_RepackedMagazinesCount">
             <English>%1 Full and %2 Partial</English>
             <French>%1 plein(s) et %2 partiel(s)</French>
-            <German>%1 Vollständigen und %2 Teilweisen</German>
+            <German>%1 Volle und %2 zum Teil gefüllte</German>
```

Comment:
Das ist mir durchaus bewusst. In der alten Übersetzung würde ich folgendes im Spiel erwarten:
3 Vollständigen und 3 Teilweisen anstelle von 3 Volle und 3 zum Teil gefüllte. 
Dementsprechend sehe ich das Problem nicht.